sdk-utils/src/main/java/com/redhat/parodos/sdkutils/SdkUtils.java

```diff
@@ -33,6 +41,66 @@ public abstract class SdkUtils {
 	private SdkUtils() {
 	}
 
+	/**
+	 * Creates and configures the APIClient using the configuration properties available
+	 * in `application.yml`
+	 * @return the ApiClient
+	 */
+	public static ApiClient getParodosAPiClient() throws ApiException, MissingRequiredPropertiesException {
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		CustomPropertiesReader reader = new CustomPropertiesReader();
+		String serverIp = reader.getServerIp();
+		String serverPort = reader.getServerPort();
+
+		if (Strings.isNullOrEmpty(serverIp) || Strings.isNullOrEmpty(serverPort)) {
+			throw new MissingRequiredPropertiesException();
+		}
+
+		int port = Integer.parseInt(serverPort);
+		if (port <= 0 && port > 65535) {
+			throw new IllegalArgumentException("serverPort must be > 0 && <= 65535");
+		}
+
+		String basePath = "http://" + serverIp + ":" + serverPort;
```

Comment:
when/should we support for https?

Reply:
TBH, I've no idea.
This is what the bash script did and, at the moment, this is a sort of porting from bash to java.
Can we discuss this and the add support for https in another PR?
Do you want me to create a Jira issue to track this?